formula-android/src/main/java/com/instacart/formula/FormulaAndroid.kt

```diff
@@ -21,10 +21,12 @@ object FormulaAndroid {
     /**
      * Initializes Formula Android integration. Should be called within [Application.onCreate].
      *
+     * @param logger A logger for debug Formula Android events.
      * @param onFragmentError A global handler for fragment errors. Override this to log the crashes.
      */
     fun init(
         application: Application,
+        logger: (String) -> Unit = {},
```

Comment:
Should it be nullable if the user doesn't care to log? and if so, maybe it should be the last param?

Reply:
I've made the `logger` nullable, however, it cannot be the last parameter because the following continuation is already occupying that spot.
```
activities: ActivityConfigurator.() -> Unit
```